chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -612,6 +612,26 @@ std::vector<Array> Split(const Array& ary, std::vector<int64_t> indices, int8_t
     return out;
 }
 
+std::vector<Array> DSplit(const Array& ary, int64_t sections) {
+    if (sections < 1) {
+        throw DimensionError("Number of sections must be larger than 0.");
+    }
+
+    if (ary.ndim() < 3) {
+        throw DimensionError("dsplit only works on arrays of 3 or more dimensions.");
+    }
+
+    return Split(ary, sections, 2);
+}
+
+std::vector<Array> DSplit(const Array& ary, std::vector<int64_t> indices) {
+    if (ary.ndim() < 3) {
+        throw DimensionError("dsplit only works on arrays of 3 or more dimensions.");
+    }
+
+    return Split(ary, indices, 2);
```

Comment:
Could you fix for cpplint?
```suggestion
    return Split(ary, std::move(indices), 2);
```

Reply:
Done :)